resources/core/charts/monitoring/charts/alertmanager/README.md

```diff
@@ -20,68 +20,105 @@ metadata:
+{{ define "alertmanager.yaml.tpl" }}
+global:
+  resolve_timeout: 5m
+route:
+  receiver: 'null'
+  group_wait: 30s
+  group_interval: 5m
+  repeat_interval: 5m
+  group_by: ['cluster','pod','job','alertname']
+  # All alerts that do not match the following child routes
+  # will remain at the root node and be dispatched to 'default-receiver'
+  routes:
+  - receiver: 'null'
+    match:
+      alertname: DeadMansSwitch
+  - receiver: "victorOps"
+    continue: true # If continue: is set to false it will stop after the first matching.
+    match_re:
+      severity: critical
+  - receiver: "slack"
+    continue: true # If continue: is set to false it will stop after the first matching.
+    match_re:
+      severity: warning|critical
+receivers:
+- name: 'null'
+- name: "victorOps"
+  victorops_configs:
+  - api_key: {{ .Values.global.alertTools.credentials.victorOps.apikey | quote }}
+    send_resolved: true
+    api_url: https://alert.victorops.com/integrations/generic/20131114/alert/
+    routing_key: {{ .Values.global.alertTools.credentials.victorOps.routingkey | quote }}
+    state_message: 'Alert: {{`{{ .CommonLabels.alertname }}`}}. Summary:{{`{{ .CommonAnnotations.summary }}`}}. RawData: {{`{{ .CommonLabels }}`}}'
+- name: "slack"
+  slack_configs:
+  - channel: {{ .Values.global.alertTools.credentials.slack.channel | quote }}
+    send_resolved: true
+    api_url: {{ .Values.global.alertTools.credentials.slack.apiurl | quote }}
+    icon_emoji: ":ghost:"
+    title: '[{{`{{ .Status | toUpper }}`}}{{`{{ if eq .Status "firing" }}`}}:{{`{{ .Alerts.Firing | len }}`}}{{`{{ end }}`}}] Monitoring Event Notification'
+    text: "<!channel> \nsummary: {{`{{ .CommonAnnotations.summary }}`}}\ndescription: {{`{{ .CommonAnnotations.description }}`}}"
+{{ end }}
 ```
 **A Quick explanation**
 * ```route:``` A route block defines a node in a routing tree and its children. Its optional configuration parameters are inherited from its parent node if not set.
 * ```routes:``` Child routes.
 * ```receiver:``` Receiver is a named configuration of one or more notification integrations.
 * ```receiver:``` A list of configured notification receivers.
 
+What this configuration provides is to enable the ```receivers:```, "victorOps" and "slack" for receiving alerts fired by Prometheus rules.
+
+In order to enable alert notifications for the receivers above, four parameters need to be configured.
+
+* ```api_key:``` defines the team Api key in VictorOps
+* ```routing_key:``` defines the team routing key in VictorOps
+* ```channel:```  the Slack channel to receive the alerts notifications
+* ```api_url:``` The url endpoint containing to send the alerts.
+
+As it was mentioned at first only part of the configuration is located in this chart, that is why all of four parameters values are got it from the template, ```{{ .Values.global.alertTools.credentials... }}```, and these values are configured in

```

Comment:
~As it was mentioned at first~ `Only a part of the configuration is located in this chart.` ~that is why~ ``All of the four parameters' values are taken from the `{{ .Values.global.alertTools.credentials... }}` template. These values are configured in [this](../../../../values.yaml) file. For example:``

Reply:
done